action.yml

```diff
@@ -6,6 +6,10 @@ inputs:
     description: 'Description regex to match'
     required: true
     default: '.*SUMMARY:.*'
+  description-regex-flags:
+    description: 'Flags for the description regex'
+    required: false
+    default: 'i'
```

Comment:
Not sure we want it to be specified here.

[**inputs.<input_id>.default**](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault)
**Optional** A string representing the default value. The default value is used when an input parameter isn't specified in a workflow file.